packages/internals/src/utils/drawBox.ts

```diff
@@ -26,7 +26,13 @@ function maxLineLength(str: string): number {
 
 export function drawBox({ title, width, height, str, horizontalPadding }: BoxOptions): string {
   horizontalPadding = horizontalPadding || 0
+  // This avoids having NaN later
+  // if these required param were not passed (in JS, for example)
```

Comment:
Let's maybe move this comment above the previous line since it applies to it as well.